webpack/assets/javascripts/react_app/components/user/passwordStrength/index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import ReactPasswordStrength from 'react-password-strength';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import CommonForm from '../../common/forms/CommonForm';
+import {
+  updatePassword,
+  checkPasswordsMatch,
+} from '../../../redux/actions/user/passwordStrength';
+import helpers from '../../../common/helpers';
+import './password_strength.scss';
+
+class PasswordStrength extends React.Component {
+  constructor(props) {
+    super(props);
+    helpers.bindMethods(this, ['checkRetype']);
+  }
+  checkRetype(event) {
+    this.props.checkPasswordsMatch(event.target.value, this.props.password);
+  }
+
+  render() {
+    const {
+      data: {
+        className, id, name, verify, error, userInputIds,
+      },
+      matchMessage,
+    } = this.props;
+
+    return (
+      <div>
+        <CommonForm label={__('Password')} touched={true} error={error}>
+          <ReactPasswordStrength
+            changeCallback={({ password }) => this.props.updatePassword(password)}
+            minLength={6}
+            minScore={2}
+            userInputs={
+              userInputIds.length > 0
+                ? userInputIds.map(input => document.getElementById(input).value)
+                : []
+            }
+            tooShortWord={__('Too short')}
+            scoreWords={[
+              __('Weak'),
+              __('Medium'),
+              __('Normal'),
+              __('Strong'),
+              __('Very strong'),
+            ]}
+            inputProps={{
+              name,
+              id,
+              className: `${className}`,
```

Comment:
do you really need the string interrelation here? I think the following would just work:
```js
inputProps={{  name,  id, className }}
```